CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 ## [NEXT RELEASE]
 
+- ROX-11443: The default value for `--include-snoozed` option of `roxctl image scan` command is set to `false`.
```

Comment:
Maybe we should also mention the behavior change, i.e. what the impact is of setting it to `false`?